src/include/access/k2/status.h

```diff
@@ -36,6 +36,7 @@ struct Status {
     std::string msg;
     std::string detail;
 
+    bool ok() const { return pg_code ==  ERRCODE_SUCCESSFUL_COMPLETION;}
```

Comment:
Already have an IsOK() method in mainline. No need for this method.

Reply:
Removed